### PR TITLE
MOBILEAPPS-2069

### PR DESCRIPTION
**Pull request by aman-alfresco**
Json parsing and model creation

### CODE REVIEW

File: app/src/main/java/com/alfresco/content/app/activity/LoginActivity.kt
Comment (gl-lovekesh) [inline app/src/main/java/com/alfresco/content/app/activity/LoginActivity.kt:1]: Aman, this change is not required, please remove them and note these changes.

File: actions/src/main/kotlin/com/alfresco/content/actions/sheet/ProcessDefinitionsSheet.kt [hunk 1]
```diff
@@ -83,7 +83,8 @@ class ProcessDefinitionsSheet : BottomSheetDialogFragment(), MavericksView {
 
                             val intent = Intent(
                                 requireActivity(),
-                                Class.forName("com.alfresco.content.browse.processes.ProcessDetailActivity"),
+                                Class.forName("com.alfresco.content.process.ProcessFormActivity"),
+//                                Class.forName("com.alfresco.content.browse.processes.ProcessDetailActivity"),
```
Comment (gl-lovekesh) [inline actions/src/main/kotlin/com/alfresco/content/actions/sheet/ProcessDefinitionsSheet.kt:87]: Why did we comment this code?

**Review by gl-lovekesh — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/alfresco/content/app/activity/LoginActivity.kt
Comment (aman-alfresco) [reply app/src/main/java/com/alfresco/content/app/activity/LoginActivity.kt:1]: Done

File: actions/src/main/kotlin/com/alfresco/content/actions/sheet/ProcessDefinitionsSheet.kt (hunk 1 above)
Comment (aman-alfresco) [reply actions/src/main/kotlin/com/alfresco/content/actions/sheet/ProcessDefinitionsSheet.kt]: It was navigation to the previous process detail controller and we used a new controller for dynamic forms so it's commented.

Comment (gl-lovekesh) [reply actions/src/main/kotlin/com/alfresco/content/actions/sheet/ProcessDefinitionsSheet.kt]: Create a "develop" branch and raise PR again for that.

**Pull request closed without merging**